prow/cmd/status-reconciler/main.go

```diff
@@ -80,7 +78,6 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	fs.BoolVar(&o.continueOnError, "continue-on-error", false, "Indicates that the migration should continue if context migration fails for an individual PR.")
 	fs.Var(&o.addedPresubmitDenylist, "denylist", "Org or org/repo to ignore new added presubmits for, set more than once to add more.")
 	fs.Var(&o.addedPresubmitDenylistAll, "denylist-all", "Org or org/repo to ignore reconciling, set more than once to add more.")
-	fs.Var(&o.addedPresubmitBlacklist, "blacklist", "[Will be deprecated after May 2021] Org or org/repo to ignore new added presubmits for, set more than once to add more.")
```

Comment:
I absolutely do not mind this being gone but if we set a later date ("after May 2021") then we should keep the promise?

Reply:
huh. Didn't notice that there is an in consistency between ANNOUNCEMENT.md and here, let's keep both promises. Thank you for pointing out!